### PR TITLE
[Docs] Improve platform.sh documentation for deployment

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.5
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

File: docs/cookbook/deployment/platform-sh.rst [hunk 1]
```diff
@@ -19,189 +13,108 @@ In this guide you will find sufficient instructions to have your application up
...
+            bin/console sylius:fixtures:load -n
+            bin/console assets:install --symlink --relative public
+            bin/console cache:clear
+
+    # The relationships of the application with services or other applications.
+    # The left-hand side is the name of the relationship as it will be exposed
+    # to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+    # side is in the form `<service name>:<endpoint name>`.
+    relationships:
+        database: "mysqldb:mysql"
```
Comment (lchrusciel) [inline docs/cookbook/deployment/platform-sh.rst]: We may add note, that `mariadb` will be installed here

File: docs/cookbook/deployment/platform-sh.rst [hunk 2]
```diff
@@ -19,189 +13,108 @@ In this guide you will find sufficient instructions to have your application up
...
-        # - { resource: parameters.yml } <- Has to be placed before our new file
-        - { resource: parameters_platform.php }
+    # The configuration of app when it is exposed to the web.
+    web:
+        locations:
+            "/":
+                # The public directory of the app, relative to its root.
+                root: "public"
+                # The front-controller script to send non-static requests to.
+                passthru: "/index.php"
```
Comment (lchrusciel) [inline docs/cookbook/deployment/platform-sh.rst]: I'm wondering if we shouldn't keep caching config for default installation https://github.com/Sylius/Sylius/pull/10916/files#diff-5366644cbd212c7b2962e464bc992943L75

File: docs/getting-started-with-sylius/deployment.rst [hunk 3]
```diff
@@ -12,123 +12,7 @@ There are many servers which you can choose for your store deployment, but in th
 Process
 -------
 
-1. Create an account on `Platform.sh <https://platform.sh/>`_
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/deployment.rst]: /cc @CoderMaggie  should we remove this part from getting started or keep it duplicated?

Comment (Tomanhez) [reply docs/getting-started-with-sylius/deployment.rst]: We talked that duplicate this is not good solution, link should be enough

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "review-platform-sh"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by GSadee** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**